README.md

```diff
@@ -11,16 +11,16 @@
 
 cder (_see-der_) is a database seeding tool to help you import fixure data in your local environment.
 
-Generating seeds programmatically is a easy task, but maintaining them is not.
+Generating seeds programmatically is an easy task, but maintaining them is not.
 Everytime when your schema is changed, your seed can be broken.
 It costs your team extra efforts to keep them updated.
```

Comment:
```suggestion
It costs your team extra effort to keep them updated.
```

Reply:
I fixed as suggested, but may I ask for an explanation in detail for future reference, if you would not mind?
For clarity, just let me tell what I thought: `efforts` here referred to humans' 'actions' required to update seeds, that is, rewriting or adding seeds data and keeping them new, and therefore `[countable] an attempt to do something especially when it is difficult to do` [1] sounded suited to me. 
[1] https://www.oxfordlearnersdictionaries.com/definition/english/effort

Reply:
That makes sense to me (actually my initial intention).
Having said that, it's not a critical point in terms of usage of this crate, so I'd accept either way (please correct if you will!) Thanks again for the detailed review 👍